src/services/connextService.ts

```diff
@@ -0,0 +1,116 @@
+import { getPoolFeeForUniV3, getXCallCallData, prepareSwapAndXCall } from '@connext/chain-abstraction';
+import { DestinationCallDataParams, Swapper, SwapAndXCallParams } from '@connext/chain-abstraction/dist/types';
+import { SdkConfig, create } from '@connext/sdk';
+import { NETWORKS, SUPPORTED_CHAINS_BY_CONNEXT } from '@constants';
+import { find } from 'lodash';
+import WalletService from './walletService';
+
+interface DomainID {
+  [key: number]: string;
+}
+
+export default class ConnextService {
+  walletService: WalletService;
+
+  sdkConfig: SdkConfig;
+
+  constructor(walletService: WalletService) {
+    this.walletService = walletService;
+    const domainConfig: { [domainId: string]: { providers: string[] } } = {};
+
+    const domainChainIds = Object.entries(SUPPORTED_CHAINS_BY_CONNEXT)
+      .filter(([key]) => typeof key === 'number')
+      .map(([key, value]) => ({ domainId: value.domainId, chainId: key }));
+
+    domainChainIds.forEach((obj) => {
+      domainConfig[obj.domainId] = { providers: [this.getRPCURL(parseInt(obj.chainId, 10))] };
+    });
+
+    const sdkConfig: SdkConfig = {
+      signerAddress: this.walletService.getAccount(),
+      network: 'mainnet', // can change it to testnet as well
```

Comment:
I woudl move this all into its own function, since walletService.getAccount() will return the current account and at the moment of the constructor the wallet is not yet connected, so its always going to be empty.

The idea would be that on the `setAccount` method of the `web3Service.ts` we call that function to regenerate the config with the new address